src/runtime/mod.rs

```diff
@@ -579,11 +589,40 @@ impl Module {
         )
         .await
     }
+    async fn load_oci(&self, uri: &Url) -> anyhow::Result<wasmtime::Module> {
+        let config = ClientConfig::default();
+        let mut oc = Client::new(config);
+        let auth = RegistryAuth::Anonymous;
+
+
+        let img = url_to_oci(uri)?;
+        let data = oc.pull(&img, &auth, vec!["application/wasm"]).await?;
```

Comment:
This should be `application/vnd.wasm.content.layer.v1+wasm`.

Reply:
Oh! Right! I forgot that there were special types for layers. I should probably refactor this into a const, too.